src/app/learn/get-started/basics/build-publish/index.html

```diff
@@ -20,17 +20,14 @@
     </li>
     <li>
       <p>
-        Enter <sky-code>skyux build</sky-code> to build the SPA into the <sky-code>dist/</sky-code> folder.
+        Run <sky-code>skyux build</sky-code> to build the SPA into the <sky-code>dist/</sky-code> folder.
       </p>
     </li>
     <li>
       <p>
-        Copy the contents of the <sky-code>dist/</sky-code> folder into a directory using the SPA name specified in <a stacheRouterLink="/learn/get-started/basics/create-project">create a SPA</a>.
+        Copy the contents of the <sky-code>dist/</sky-code> folder into a directory with the SPA name that you specified when you <a stacheRouterLink="/learn/get-started/basics/create-project">created the SPA</a>. For example, if you named the SPA <sky-code>my-spa</sky-code> when you ran <sky-code>skyux new</sky-code>, change the <sky-code>dist/</sky-code> folder to <sky-code>my-spa/</sky-code>.
       </p>
-      <p>
-          For example, if you provided <sky-code>my-spa</sky-code> after running <sky-code>skyux new</sky-code>, change the <sky-code>dist/</sky-code> folder to <sky-code>my-spa/</sky-code>.
-      </p>
-      <sky-alert alertType="info">If you would like to host your SPA at the root, without having to use a subfolder, you can set the <a routerLink="/learn/reference/configuration/apply-appsettings">base property</a> in <sky-code>skyuxconfig.json</sky-code> to <sky-code>/</sky-code>.</sky-alert>
+      <sky-alert alertType="info">To host your SPA at the root without a subfolder, set the <sky-code>base</sky-code> property to <sky-code>/</sky-code> in <a routerLink="/learn/reference/configuration" fragment="app">the <sky-code>app</sky-code> configuration option</a> in <sky-code>skyuxconfig.json</sky-code>.</sky-alert>
```

Comment:
Can we say section instead of option?

Reply:
We should if you think that's clearer or more accurate. ... But FWIW, we use "option" elsewhere, so this would be a tweak to the terminology we currently use. For example, we use "option" in the warning at the end of this topic. But if "section is more accurate, I can change both and make a note to update it elsewhere too.

Reply:
I think the reason those feel different are because of the type of values they support.  

In the case of `app`, it's only valid values are child properties/options, which is why I suggested "section."

In the case of `base`, it's only valid values are strings.

Reply:
I updated it to "section."